update-travis-config.sh

```diff
@@ -24,22 +24,18 @@ print_static_configuration() {
 	  - docker
 	before_install:
 	  - ./update-docker-files.sh
+	  - ./update-img.sh
 	  - docker info
 	  - docker run --rm --privileged multiarch/qemu-user-static:register --reset
 	install:
 	  - docker build --build-arg VCS_REF=$TRAVIS_COMMIT --build-arg BUILD_DATE=$(date +"%Y-%m-%dT%H:%M:%SZ") --build-arg VERSION=$VERSION -t $DOCKER_REPO:$VERSION-$TARGET-$DIST $VERSION/$TARGET/$DIST
 	  - docker run --rm $DOCKER_REPO:$VERSION-$TARGET-$DIST uname -a
 	after_success:
-	  - docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD
-	  - docker push $DOCKER_REPO:$VERSION-$TARGET-$DIST
+	  - if [[ "$TRAVIS_BRANCH" == "master" ]]; docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD; fi
+	  - if [[ "$TRAVIS_BRANCH" == "master" ]]; docker push $DOCKER_REPO:$VERSION-$TARGET-$DIST; fi
```

Comment:
Shouldn't you replace all occurrences of `docker` with `img` so it is actually used and not just downloaded?

Reply:
The script update_img.sh and the loading of img is not mine!
I would first test it in a separate PR.

Reply:
If the `img` command remains unused in this PR, it would be better to add `update_img.sh` and update `.travis.yml` in the separate PR where `img` is used instead of `docker`. :wink:

Reply:
Is the `if [[ "$TRAVIS_BRANCH" == "master" ]];` check for the master branch even necessary? 

.travis.yml already contains:

https://github.com/openhab/openhab-docker/blob/73bac30cabb1808854d08a6c45309348c73ae001/.travis.yml#L10-L12

See also the [Travis documentation](https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches).

Reply:
It also results in errors according to the Travis build log:

```
travis_fold:start:after_success.1
[0Ktravis_time:start:21c512d3
[0K$ if [[ "$TRAVIS_BRANCH" == "master" ]]; docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD; fi
/home/travis/.travis/job_stages: eval: line 104: syntax error near unexpected token `fi'
/home/travis/.travis/job_stages: eval: line 104: `if [[ "$TRAVIS_BRANCH" == "master" ]]; docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD; fi '
travis_time:end:21c512d3:start=1542586679094465625,finish=1542586679099209151,duration=4743526
[0Ktravis_fold:end:after_success.1
[0Ktravis_fold:start:after_success.2
[0Ktravis_time:start:0e38e176
[0K$ if [[ "$TRAVIS_BRANCH" == "master" ]]; docker push $DOCKER_REPO:$VERSION-$TARGET-$DIST; fi
/home/travis/.travis/job_stages: eval: line 104: syntax error near unexpected token `fi'
/home/travis/.travis/job_stages: eval: line 104: `if [[ "$TRAVIS_BRANCH" == "master" ]]; docker push $DOCKER_REPO:$VERSION-$TARGET-$DIST; fi '
travis_time:end:0e38e176:start=1542586679105175562,finish=1542586679108541526,duration=3365964
```

Reply:
Yes you are right. I did not notice that at all.